controllers/admin/projects/projects.controller.js

```diff
@@ -0,0 +1,30 @@
+const model = require('../../../models/index')
+
+// Return all projects
+exports.findAll = (req, res) => {
+  model.Project.findAll()
+    .then(projects => {
+      res.send(projects);
+    })
+    .catch(err => {
+      res.status(500).send({
+        message:
+          err.message || "Some error occurred while retrieving projects."
+      });
+    });
+}
+
+// Return information about a single project
+exports.findOne = (req, res) => {
+  model.Project.findByPk(req.params.id)
+  .then(project => {
+    if (project == null) {
+      res.status(404).send({ message: `Project with id: ${req.params.id} does not exist` });
+    } else {
+      res.send(project);
+    }
+  })
+  .catch(err => {
+    res.status(500).send({ message: err.message || 'Something went wrong' });
+  })
+}
```

Comment:
Does return the projects but again, shouldn't his own projects maybe be highlighted?

Reply:
I'd say it's implied based on the route the admin will be making a request to. Don't think we can really add much to differentiate the difference